lib/unzip.js

```diff
@@ -0,0 +1,13 @@
+export function zip() {
+  let array = Array.prototype.slice.call(arguments);
+  let length = array[0].length;
+  let ar = [];
+  for (let i = 0; i < length; i++) {
+    let result = array.reduce(function(acc, item, index, arr) {
```

Comment:
`result` jest `const` a przynajmniej powinien byc

Reply:
Jak nie używasz parametrów `index, arr` to nie musisz deklarować że funkcja je przyjmuje